tests/components/google_assistant/test_google_assistant.py

```diff
@@ -93,6 +99,16 @@ def hass_fixture(loop, hass):
         ceiling_lights_entity.state,
         attributes=attrs)
 
+    # By setting the google_assistant_type = 'light'
```

Comment:
Don't do this. Just set it in the state machine only for the test that is going to use it. No need to override an existing entity.